src/modes/map/map_treasure.cpp

```diff
@@ -318,9 +318,19 @@ void TreasureSupervisor::_UpdateList()
             // If true, the drunes have been selected
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
-            _detail_textbox.SetDisplayText(VTranslate("With the additional %u drunes found in this treasure added, "
-                                                      "the party now holds a total of %u drunes.",
-                                                      _treasure->_drunes, GlobalManager->GetDrunes()));
+
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
> Indentation is such a nightmare with QTCreator,

I'm using qtcreator atm and it is really fine with the 4-spaces rule so far. I went to tools->options->text editor->behavior and I ensured the tab policy was on spaces only + indentation size set to 4.

> The comment is for the translators, not for the developers. It will be easier for translators to find with the whole sentence in it.

Well, of course, my point was about to use some common way to use that feature both in lua and in c++. ;)